elements/lisk-p2p/src/p2p.ts

```diff
@@ -124,6 +124,7 @@ export const DEFAULT_POPULATOR_INTERVAL = 10000;
 export const DEFAULT_SEND_PEER_LIMIT = 25;
 // Max rate of WebSocket messages per second per peer.
 export const DEFAULT_WS_MAX_MESSAGE_RATE = 100;
+export const DEFAULT_WS_MAX_MESSAGE_RATE_PENALTY = 10;
```

Comment:
Should penalty be `100`? Otherwise the actual number of messages allowed would be 10x

Reply:
@mitsujutsu `DEFAULT_WS_MAX_MESSAGE_RATE` means the rate above which we disconnect, not ban. Banning is more serious; we want to restrict it to intentional DoS attempts, not one-offs. For example, if someone is running a node which has bad configs or custom logic, we want to give the operator of that node a chance to fix their configs before they are banned from the network.

Reply:
I see. So in practice they are allowed to exceed the max rate 10 times before they are banned?